pkg/api/api.proto

```diff
@@ -460,6 +468,20 @@ message GetTrialsReply {
     repeated Trial trials = 1;
 }
 
+/**
+ * Get a trial configuration from DB by trial ID
+ */
+message GetTrialRequest {
```

Comment:
This request may be confusing with GetTrialsRequest above, since the names differ only by 1 letter and the parameters are the same. Maybe we should rename GetTrialsRequest to ListStudyTrialsRequest? @YujiOshima What do you think?

Reply:
Ok, I can change it in another PR.
BTW, should we keep backward compatibility about GetTrialsRequest

Reply:
Perhaps we should reconsider this when we define our Beta APIs.  For now this looks good.